### DISCUSSION TITLE
Ansible 8 requires something called an "AnsibleCollectionFinder"

### DISCUSSION THREAD

**Discussion by l0b0** [ansible/ansible-lint#3520] [Q&A] [answered]
A Dependabot update from Ansible 7.6.0 to 8.0.0 is breaking because ansible-lint now complains that

> [A]n AnsibleCollectionFinder has not been installed in this process

How do I deal with this? Google doesn't seem to know what an "Ansible collection finder" is, and I can't find it in the Ansible Lint documentation. The only reference I can find is the [actual code](https://github.com/ansible/ansible/blob/79677c16f175e52a0dee9f2d366775e8ed0c8231/lib/ansible/utils/collection_loader/_collection_config.py#L97), which isn't super helpful to figure out what to do next.

**Comment by sennewood** [comment]
I have the same issue with 8.1.0

**Reply by bryder** [reply, marked as answer]
My problem may not match yours.

In my case I'm on Ubuntu 22.04 and  ansible-lint gives the same error.

I fixed it by pip installing the latest ansible lint (6.20.3). 

The problem seems to be the fact we have python 3.10 which is not compatible with ansible-lint 5.4.0
